source/renderer/app/containers/wallet/WalletReceivePage.tsx

```diff
@@ -70,7 +70,13 @@ class WalletReceivePage extends Component<Props, State> {
     const dialog = WalletReceiveDialog;
 
     if (activeWallet && activeWallet.isHardwareWallet) {
-      hardwareWallets.initiateAddressVerification(addressToShare);
+      try {
+        hardwareWallets.initiateAddressVerification(addressToShare);
+      } catch (err) {
+        hardwareWallets.resetInitializedAddressVerification({
```

Comment:
I think `reset` makes more sense because this method is used as a way to clean up states and variables. This doesn't necessarily means the address flow was completed.